e2e/updatecli.d/success.d/hcl/scm.yaml

```diff
@@ -0,0 +1,58 @@
+name: Test HCL resources
+
+scms:
+  default:
+    kind: git
+    spec:
+      url: https://github.com/jenkins-infra/aws.git
```

Comment:
Probably should be changed to somewhere in https://github.com/updatecli-test

Reply:
Definitely, I now try to move test repository to updatecli-test systematically.
I just fork it to https://github.com/updatecli-test/jenkins-infra-aws